gitbom/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 use gitoid::GitOid;
+use gitoid::ObjectType::Blob;
```

Comment:
Although a warning is generated that this import is not used, it's definitely used whenever we use a Blob as an object type that is passed to a GitOid API call. I'm not sure why it is generating the warning.

Reply:
If the import is just in the test module, then it's unused in the parent module. I think if you move the import to the test module this will go away.

Reply:
That make sense, fixing!

Reply:
Done!